### PR TITLE
test: added unit tests

### LINKED ISSUE #13: add unit test
should test:
-service entity to model conversion
-service model to entity conversion
-existence validation on add/delete/update

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
closes #13

### CODE REVIEW

File: src/DAL/ConnectionManager.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  private connection?: Connection;
+
+  public constructor(
+    @inject(delay(() => MCLogger)) private readonly logger: MCLogger
+  ) {}
+
+  public async init(): Promise<void> {
+    const connectionConfig = config.get<ConnectionOptions>('typeOrm');
+    try {
+      this.connection = await createConnection(connectionConfig);
```
Comment (asafMasa) [inline src/DAL/ConnectionManager.ts:24]: add info log about connecting including connection params

File: src/DAL/ConnectionManager.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import {
```
Comment (asafMasa) [inline src/DAL/ConnectionManager.ts:1]: don't use PascalCase for file names

File: src/controllers/ImagesController.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import { Request, Response } from 'express';
```
Comment (asafMasa) [inline src/controllers/ImagesController.ts]: don't use PascalCase for file names

File: src/services/ImageService.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import { injectable } from 'tsyringe';
```
Comment (asafMasa) [inline src/services/ImageService.ts]: don't use PascalCase for file names

File: src/DAL/ImageDataRepository.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import { Repository, EntityRepository } from 'typeorm';
```
Comment (asafMasa) [inline src/DAL/ImageDataRepository.ts]: Don't use PascalCase for file names

File: src/entity/ImageData.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm';
```
Comment (asafMasa) [inline src/entity/ImageData.ts]: Don't use PascalCase for file names

File: src/middleware/ErrorHandler.ts [hunk 7]
```diff
@@ -1,7 +1,9 @@
 import { MCLogger } from '@map-colonies/mc-logger';
```
Comment (asafMasa) [inline src/middleware/ErrorHandler.ts]: Don't use PascalCase for file names

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job
🥇

**Review by asafMasa — APPROVED** [review]

File: src/DAL/ConnectionManager.ts (hunk 1 above)
Comment (cl-shaharshavit-rnd) [reply src/DAL/ConnectionManager.ts:24]: added

**Pull request merged**